packages/cli/generators/relation/relationutils.js

```diff
@@ -0,0 +1,158 @@
+'use strict';
+
+const utils = require('../../lib/utils');
+
+exports.relationType = {
+  belongsTo: 'belongsTo',
+  hasMany: 'hasMany',
+  hasOne: 'hasOne',
+};
+
+exports.addFileToProject = function(project, path, modelName) {
+  const fileName = path + '/' + modelName + '.model.ts';
+  return project.addExistingSourceFile(fileName);
+};
+
+exports.getClassObj = function(fileName, modelName) {
+  return fileName.getClassOrThrow(modelName);
+};
+
+exports.getClassesCount = function(fileName) {
+  return fileName.getClasses().length;
+};
+
+exports.isPropertyExist = function(classObj, propertyName) {
```

Comment:
`isPropertyExist` does not read like English to me. How about `modelHasProperty`, `hasProperty` or `propertyExists`?

I see that you are using names starting with `does` below, `doesPropertyExist` would be a good name too.

Reply:
Fixed.